api/api/projects_api_test.go

```diff
@@ -369,14 +376,64 @@ func TestListProjects(t *testing.T) {
 
 func TestUpdateProject(t *testing.T) {
 	testCases := []struct {
-		desc             string
-		projectID        models.ID
-		existingProject  *models.Project
-		expectedResponse *Response
-		body             interface{}
+		desc                  string
+		projectID             models.ID
+		existingProject       *models.Project
+		expectedResponse      *Response
+		expectedMessage       map[string]interface{}
+		body                  interface{}
+		updateProjectEndpoint string
+		updateProjectPayload  string
+		updateProjectResponse string
```

Comment:
These two names are misleading. They don't actually refer to the actual payload or response right?